src/ImageStorage.php

```diff
@@ -22,10 +22,10 @@ class ImageStorage
 	/** @var string */
 	private $data_dir;
 
-	/** @var string */
+	/** @var callable(string): string */
```

Comment:
Interesting syntax. Where is it from? I found it only in [psalm docs](https://github.com/vimeo/psalm/blob/master/docs/typing_in_psalm.md#callables-and-closures)

Reply:
i think originally from typescript, phpstan supports this some time, but it is undocumented

Reply:
it was implemented in 0.10.3
https://github.com/phpstan/phpstan/releases/tag/0.10.3